### ISSUE TITLE
Veldrid-spirv for MacCatalyst

### ISSUE THREAD

**Issue by dtatarnikov** [frenzibyte/MetalSandbox#1]
I've tried to launch MacCatalyst application to work with veldrid-spirv with your wrapper of xcframework. But you do not have MacCatalyst as one of the compiling targets, so it can not be launched successfully. Is there any possibility to add MacCatalyst as a target in veldrid-spirv.framework for arm64 and x86-64 architechtures?
P.S.: I would like to add this issue in ppy/veldrid-spirv or your fork, but issues are closed for these repositories.

**Comment by frenzibyte** [comment]
Open an issue thread at https://github.com/veldrid/veldrid-spirv

**Comment by dtatarnikov** [comment]
> Open an issue thread at https://github.com/veldrid/veldrid-spirv

Thanks for reply. As you know [mellinoe](https://github.com/veldrid/veldrid-spirv/commits?author=mellinoe) stop to maintain this repository, and new issues are not closed for a long time. Moreover, I wrote about wrapper  framework "veldrid-spirv.framework" that you've been using here in project (the original source code use static library). I've tried "veldrid-spirv.framework" and it works ok with iPhones and simulators, so I've been able to launch veldrid-spirv (against the original static library, which do not work for me). 
If you have source code for "veldrid-spirv.framework" and may share it, I would able to compile it against MacCatalyst and try to use it. It will helped a lot with my project.
